rocksdb_admin/application_db_manager.cpp

```diff
@@ -124,6 +124,15 @@ std::string ApplicationDBManager::DumpDBStatsAsText() const {
   return stats;
 }
 
+const std::vector<std::string> ApplicationDBManager::getAllDBNames()  {
+    std::shared_lock<std::shared_mutex> lock(dbs_lock_);
+    std::vector<std::string> db_names;
```

Comment:
move it to the first line of this function, and call a reserve(dbs_.size()) after lock(dbs_lock_)